recipes/llvm/all/conanfile.py

```diff
@@ -0,0 +1,128 @@
+from conans import ConanFile, tools, CMake
+from conans.tools import Version
+from conans.errors import ConanInvalidConfiguration
+import os, shutil, glob
+
+projects = [
```

Comment:
[1/2] Why didn't you split projects and runtimes (more on this below)?
https://llvm.org/docs/CMake.html#frequently-used-llvm-related-variables

Namely:

> LLVM_ENABLE_PROJECTS:STRING
>     Semicolon-separated list of projects to build, or all for building all (clang, lldb, lld, polly, etc) projects. This flag assumes that projects are checked out side-by-side and not nested, i.e. clang needs to be in parallel of llvm instead of nested in llvm/tools. This feature allows to have one build for only LLVM and another for clang+llvm using the same source checkout. The full list is: clang;clang-tools-extra;cross-project-tests;libc;libclc;lld;lldb;openmp;polly;pstl
> LLVM_ENABLE_RUNTIMES:STRING
>     Build libc++, libc++abi, libunwind or compiler-rt using the just-built compiler. This is the correct way to build runtimes when putting together a toolchain. It will build the builtins separately from the other runtimes to preserve correct dependency ordering. If you want to build the runtimes using a system compiler, see the [libc++ documentation](https://libcxx.llvm.org/BuildingLibcxx.html). Note: the list should not have duplicates with LLVM_ENABLE_PROJECTS. The full list is: compiler-rt;libc;libcxx;libcxxabi;libunwind;openmp To enable all of them, use: LLVM_ENABLE_RUNTIMES=all

Reply:
For some context, I was just building LLVM, and this is what I used:
```sh
cmake -S llvm -B build -G Ninja -DLLVM_ENABLE_PROJECTS='clang;clang-tools-extra;libclc;lld;lldb;polly;pstl' -DLLVM_ENABLE_RUNTIMES='libcxx;openmp'  -DLLVM_PARALLEL_LINK_JOBS=2 -DLLVM_PARALLEL_COMPILE_JOBS=8 -DCMAKE_BUILD_TYPE=Release  -DCMAKE_INSTALL_PREFIX=D:\Programs\LLVM_trunk 
```

Reply:
Thanks for this. I brought your suggested changes in.